integration_tests/CMakeLists.txt

```diff
@@ -330,7 +352,7 @@ RUN(NAME test_builtin_round  LABELS cpython llvm c)
 RUN(NAME test_builtin_divmod LABELS cpython llvm c)
 RUN(NAME test_builtin_sum    LABELS cpython llvm c)
 RUN(NAME test_math1          LABELS cpython llvm c)
-RUN(NAME test_math_02        LABELS cpython llvm)
+RUN(NAME test_math_02        LABELS cpython_3_11 llvm)
```

Comment:
This will disable this for current Python, which I would not do.

Split just the functionality that needs Python 3_11 and perhaps put into one file.

Reply:
Yup, @harshsingh-24 create a separate file and add the cbrt and exp2 tests there and enable cpython_3_11 for it. Leave cpython for the rest of the tests.

Reply:
We will not add this new option `cpython_3_11`, see: https://github.com/lcompilers/lpython/pull/1594#issuecomment-1477227858